### PR TITLE
rooster: update to 2.9.0.

### PR DESCRIPTION
**Pull request by uw2021**
#### General
- [ ] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [ ] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [x] I generally don't use the affected packages but briefly tested this PR

### CODE REVIEW

File: srcpkgs/rooster/template [hunk 1]
```diff
@@ -1,17 +1,17 @@
 # Template file for 'rooster'
 pkgname=rooster
-version=2.8.6
+version=2.9.0
 revision=1
 build_style=cargo
 hostmakedepends="pkg-config python3"
-makedepends="libX11-devel libXmu-devel libressl-devel libsodium-devel"
+makedepends="libX11-devel libXmu-devel libressl-devel libsodium-devel libssl48"
```
Comment (ericonr) [inline srcpkgs/rooster/template:7]: Why did you add this?? The correct `libssl` will be pulled in by `libressl-devel`.

File: srcpkgs/rooster/template [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
 maintainer="Ulf <void@uw.anonaddy.com>"
 license="Apache-2.0"
 homepage="https://github.com/conradkdotcom/rooster"
 distfiles="https://github.com/conradkdotcom/rooster/archive/v${version}.tar.gz"
-checksum=b05d2c368ac26d96edd058130bfc7cc7a1960a5bfbc63b6ddfad62d27b08d62f
+checksum=2cb08c90c9604951c810038dffd1dfca1de71bfe1d76b22f951dc3f14c0efab4
 
 pre_build() {
-	cargo update --package openssl-sys --precise 0.9.58
+	cargo update --package openssl-sys
```
Comment (ericonr) [inline srcpkgs/rooster/template:16]: Either completely remove this or leave the precise update.

Comment (uw2021) [reply srcpkgs/rooster/template]: Couldn't get it to compile with that flag before. Same goes for libssl. Compiled fine now though.

**Comment by ericonr** [comment]
Please remember to have a single commit per package.
